src/conventional/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+list(APPEND programs
+  metar_csv2ioda.py
+)
+
+file( RELATIVE_PATH SCRIPT_LIB_PATH ${CMAKE_BINARY_DIR}/bin ${PYIODACONV_BUILD_LIBDIR} )
+conf_targets_deps( "${programs}"
+                   ${CMAKE_CURRENT_SOURCE_DIR}
+                   ${CMAKE_BINARY_DIR}/bin
+                   bin_metar_scripts_deps)
+execute_process(COMMAND chmod +x ${bin_metar_scripts_deps})
```

Comment:
You shouldn't need to run chmod yourself. If the input file has execute permission (and this gets recorded in git), the output file should, too.

Reply:
I mimicked something else in the file, so I can easily remove this.

Reply:
I eliminated the line.